src/radical/pilot/pmgr/launching/base.py

```diff
@@ -421,6 +421,13 @@ def _start_pilot_bulk(self, resource, schema, pilots):
 
```

Comment:
I did not used - 
1. pathlib - because I did not find in nearby code, probably we don't use this 
2. sh_callout - I can use this if you say so, but didn't find a good reason to do so. 
3. ru - I did not find any particular method for checking file exists in utils 
@andre-merzky

Reply:
`os.path` is perfectly acceptable for local files.